frontend/packages/console-app/src/models/index.ts

```diff
@@ -1,27 +1,14 @@
 import { K8sKind } from '@console/internal/module/k8s';
 
 export const WorkspaceModel: K8sKind = {
-  kind: 'Workspace',
-  label: 'Workspace',
-  labelPlural: 'workspaces',
-  apiGroup: 'workspace.che.eclipse.org',
-  apiVersion: 'v1alpha1',
-  abbr: 'WS',
-  namespaced: true,
-  crd: true,
-  plural: 'workspaces',
-  propagationPolicy: 'Foreground',
-};
-
-export const DevWorkspaceModel: K8sKind = {
   kind: 'DevWorkspace',
-  label: 'Devworkspace',
-  labelPlural: 'workspaces',
-  apiGroup: 'apiextensions.k8s.io',
+  label: 'DevWorkspace',
+  labelPlural: 'devworkspaces',
+  apiGroup: 'workspace.devfile.io',
   apiVersion: 'v1alpha1',
   abbr: 'DW',
   namespaced: true,
   crd: true,
   plural: 'devworkspaces',
-  propagationPolicy: 'Foreground',
+  propagationPolicy: 'Background',
```

Comment:
We should determine what isn't getting garbage collected. This doesn't necessarily fix the underlying problem since there's still some resource that isn't deleted.

Reply:
It looks to be related to https://github.com/kubernetes/kubernetes/issues/90512

Reply:
Created issue https://github.com/eclipse/che/issues/17146 with more details, but I still have no idea what resource isn't being cleaned up here -- all owned objects are removed but e.g. the deployment is stuck with a `foregroundDeletion` finalizer.

Reply:
Figured this one out -- the workspace controller webhooks were blocking the removal of the `foregroundDeletion` finalizer. This is fixed by https://github.com/devfile/devworkspace-operator/pull/107.

Is there a preference between foreground and background deletion here? The default for cli tools is `background`.